Hardware/Hardware/Printer/ReceiptParameters.swift

```diff
@@ -0,0 +1,19 @@
+/// Encapsulates the information necessary to print a receipt for a
+/// card present payment
+public struct ReceiptParameters {
```

Comment:
Maybe `CardPresentReceiptParameters` instead - we might want to keep the generic ReceiptParameters available for the future

Reply:
Sounds good